docs/source/changelog.md

```diff
@@ -4,9 +4,49 @@
 
 ## [Unreleased]
 
+## 3.0
+
+### [3.0.0] - 2022-03-10
+
```

Comment:
```suggestion

This release replaces an synchronous Kubernetes client library with an async alternative, allowing the use of native Python async features.

```
If we're keeping the following paragraphs I think a short straight-to-the-point summary is good to have.